packages/udaru-core/lib/ops/organizationOps.js

```diff
@@ -126,7 +126,7 @@ function buildOrganizationOps (db, config) {
         id, name, description, metadata
       )
       VALUES (
-        ${id}, ${name}, ${description}, ${metadata} 
+        ${id}, ${name}, ${description}, ${metadata || null} 
```

Comment:
What was the stored data with `metadata` being `undefined` in the previous version?

Reply:
I got fields with literally `undefined` in them, and throwing error from v1.3.1.
Since they are nullable in db, following the suggestion in the docs of `@nearform/sql` by replacing them with `null`.